src/components/common/TechnologyLogos/styles.js

```diff
@@ -0,0 +1,33 @@
+export default theme => ({
+  root: {
+    marginTop: 40
+  },
+  brand: {
+    width: '50%',
+    display: 'block',
+    textAlign: 'center',
+    float: 'left',
+    marginBottom: 5,
+    '& img': {
+      diplay: 'block'
+    }
+  },
+  name: {
+    fontFamily: theme.typography.secondary,
```

Comment:
Also, use CSS ellipsis to prevent text overflowing.

Reply:
in this case it is unnecessary since the text with flexbox adapts to several lines according to the content length